packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-services/TerraService.tool.mdx

```diff
@@ -62,6 +62,29 @@ export.config = {
 };
 ```
 
+### selector
+
+An optional selector can be configured to specify the element to be captured in the screenshot. The default selector is `data-terra-test-content`. Accessibility testing ignores this option and always tests the entire document.
+
+Type: `string`
+
+Required: `false`
+
+Default: `[data-terra-test-content] *:first-child`
+
+Example:
+
+```js
+// wdio.conf.js
+export.config = {
+    // ...
+    serviceOptions: {
```

Comment:
doc wise, i think we could split the constructor options and serviceOptions into separate sections